var/ramble/repos/builtin/package_managers/pip/package_manager.py

```diff
@@ -238,10 +238,10 @@ def get_package_list(self, workspace):
 
     def environment_load_commands(self):
         self.runner.configure_env(self.app_inst.expander.env_path)
-        return [self.runner.generate_activate_command()]
+        return self.runner.generate_activate_command()
```

Comment:
Super annoying nit, but can you move this to a separate PR? This way we can track that these were broken in the pip package manager in a different merge.